modules/authentication/components/ForgotPassword.jsx

```diff
@@ -0,0 +1,54 @@
+import React, {
+  cloneElement,
+  Component,
+  PropTypes
+} from 'react';
+import mapFormValues from '../utils/mapFormValues';
+
+const propTypes = {
+  actions: PropTypes.shape({
+    authentication: PropTypes.shape({
+      requestPasswordReset: PropTypes.func.isRequired
+    }).isRequired
+  }).isRequired,
+  children: PropTypes.node,
+  router: PropTypes.shape({
+    replace: PropTypes.func.isRequired
+  })
+};
+
+class ForgotPassword extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const values = mapFormValues(this.forgotPasswordForm.elements);
+
+    return this.props.actions.authentication
+      .requestPasswordReset(values)
+      .then(() => {
+        this.props.router.replace({ pathname: '/' });
```

Comment:
Haha no pardon needed. But yeah, I think it's just what we've been doing. After a password reset the user will be logged in so that's why I didn't send the user to `/sign-in`. Do you think there's a better place for the redirect? A confirmation screen?